cookbooks/adiabatic_boundary.prm

```diff
@@ -0,0 +1,80 @@
+set Dimension                              = 3
+
+set Use years in output instead of seconds = true
+set End time                               = 0
+set Nonlinear solver scheme                = single Advection, single Stokes
+set Nonlinear solver tolerance             = 1e-5
+set Linear solver tolerance                = 1e-5
+set Max nonlinear iterations               = 10 
+set Temperature solver tolerance           = 1e-4
+set Number of cheap Stokes solver steps    = 0
```

Comment:
In this case, the model should converge much faster when you use lots of `cheap Stokes solver steps`. I suggest changing this value to something like 2000. You will also need to move this parameter into the 'Stokes solver parameters' subsection.